ui/src/components/header/RoleTabs.js

```diff
@@ -43,6 +43,10 @@ class RoleTabs extends React.Component {
             label: 'History',
             name: 'history',
         },
+        {
```

Comment:
let's keep the ordering of the tabs consistent with the ordering of the columns in RoleTable

Reply:
not sure what do you mean

Reply:
in the domain page with roleTable, we have "tags" column after "policy-rule" and before "settings".